src/main/java/org/symphonyoss/symphony/messageml/elements/TextArea.java

```diff
@@ -91,4 +103,68 @@ public String getElementId(){
     return MESSAGEML_TAG;
   }
 
+  /**
+   * This method checks if the values assigned to minlength and maxlength attributes
+   * are valid. If there is an initial value in the textarea it also checks if the latter is
+   * between the range given
+   *
+   * @throws InvalidInputException when the attributes value are not valid or the input is not in range
+   */
+  private void validateMinAndMaxLengths() throws InvalidInputException {
```

Comment:
I suppose that this method (and the following?) has been copied directly from TextField
What about use the same pattern I implemented in RegexElement, LabelableElement, TooltipableElement... ?
You can create an interface (e.g. MixMaxLengthElement ?), implement this method inside as default method and add it to TextArea.

Reply:
Done in the next commit!